lib/editing/editor_codemirror.css

```diff
@@ -53,12 +53,15 @@
 .CodeMirror-hint {
   background: #3f3f3f;
   color: #dcdccc;
-  border-radius: 0px;
+  border-radius: 0;
+  max-width: 30em; /* overrides the default 19em in show-hint.css */
+  padding-right: 10px;
 }
 
 li.CodeMirror-hint-active {
   background: #545454;
   color: #f0dfaf;
+  max-width: none;
```

Comment:
It's a little jarring that the box resizes. They _do_ get to see more of the completion info. I think on the whole having the popup stay the same size, even given the tradeoff, seems like a smoother UX.